documentation/definition-of-done-other.md

```diff
@@ -0,0 +1,6 @@
+<!-- @license CC0-1.0 -->
+
+# Definition of done - other
```

Comment:
Is het nodig om dit nu al uit te splitsen over meerdere documenten? Ik denk dat ik een voorkeur heb voor één document over de DOD met tussenkopjes.